haproxy/datadog_checks/haproxy/config_models/instance.py

```diff
@@ -44,6 +44,14 @@ class Config:
     target_metric: Optional[TargetMetric]
 
 
+class Metric(BaseModel):
```

Comment:
HAProxy model was failing on master, `ddev validate models` is not ran on master so we did not catch it.